modules/environment-roles/templates/shared_reference_generator_policy.json.tpl

```diff
@@ -9,7 +9,9 @@
       ],
       "Resource": [
         "arn:aws:iam::${account_id}:policy/DaReferenceCounterTableAccessPolicy${title(environment)}",
-        "arn:aws:iam::${account_id}:role/da*"
+        "arn:aws:iam::${account_id}:role/da*",
+        "arn:aws:iam::${account_id}:policy/DAReferenceGeneratorLambdaPolicy${title(environment)}",
+        "arn:aws:iam::${account_id}:policy/da-reference-generator-${title(environment)}-vpc-policy"
```

Comment:
Shouldn't be using the `title` function here. Should be: arn:aws:iam::${account_id}:policy/da-reference-generator-${environment}-vpc-policy